src/scripts/gym/Gym.ts

```diff
@@ -73,7 +73,11 @@ class Gym extends TownContent {
         super(requirements);
         this.flags.quest = quest;
         this.flags.achievement = achievement;
-        this.buttonText = leaderName.replace(/\d/g,'');
```

Comment:
We should probably continue to replace the `\d` as we have `Bruno2` (and maybe others, not too sure)

Reply:
Yeah, i am stupid. Did not think it was used for anything.
It's back now